docs/custom_promise_types/cfengine.py

```diff
@@ -137,6 +137,9 @@ def _handle_validate(self, promiser, attributes):
         except ValidationError as e:
             self.log_error(e)
             self._result = Result.INVALID
+        except Exception as e:
+            self.log_error(f"{type(e).__name__}: {e}")
+            self._result = Result.INVALID
```

Comment:
According to the spec, I think this should be log level critical, and the result should be error:

https://github.com/cfengine/core/blob/master/docs/custom_promise_types/modules.md#results

Reply:
> Makes sense, one comment about following the spec, but not super critical.
>> According to the spec, I think this should be log level critical, and the result should be error:

It's not super critical, but it should be critical :laughing: \
I will fix !